maker-core/atlas.md

```diff
@@ -0,0 +1,45 @@
+# Atlas Immutable Alignment Artifact
+
+The Atlas Immutable Alignment Artifact (Atlas) is the foundational set of rules that govern MakerDAO. It provides a high-level description of the DAO, its actors, incentives, and work areas. Together with the Scope Bounded Mutable Alignment Artifacts (Scope Artifacts), the Atlas defines the operational framework of the DAO.
+
+The Atlas is currently unfinished but active. Once the Endgame State is reached, the Atlas will become immutable.
+
```

Comment:
I would add a link to the Atlas on the MIPs portal

Reply:
Maybe to the Scopes too (maker-core/scopes-and-artifacts.md)